app/services/vvs/vvsDateConverters.js

```diff
@@ -0,0 +1,56 @@
+function apiDateConverter(date) {
+  console.log("here");
+  const apiYear = date.getFullYear();
+  const apiMonth = () => {
```

Comment:
duplicate code for date.getMonth | date.getDate | date.getHours | date.getMinutes
Is this solvable using a parameter?